### PR TITLE
Add no argument group command syntax

### PR DESCRIPTION
**Pull request by ZakShearman**
Adds a syntax for /group when there are no arguments.

If the user does not have a group or they are alone in their group, it gives a message prompt to invite another player. Otherwise, it lists the party members.

### CODE REVIEW

File: src/main/java/net/minestom/arena/group/GroupCommand.java [hunk 1]
```diff
@@ -15,6 +21,33 @@ public GroupCommand() {
         super("group");
         setCondition(CommandUtils::lobbyOnly);
 
+        addSyntax((sender, context) -> {
+            if (sender instanceof Player player) {
```
Comment (TheMode) [inline src/main/java/net/minestom/arena/group/GroupCommand.java:25]: The player check is unnecessary. As already made by `CommandUtils::lobbyOnly`

Comment (ZakShearman) [reply src/main/java/net/minestom/arena/group/GroupCommand.java:25]: Ahh I just did so because the one below did as well. I'll change it

Comment (TheMode) [reply src/main/java/net/minestom/arena/group/GroupCommand.java:25]: Will have to be fixed too

Comment (ZakShearman) [reply src/main/java/net/minestom/arena/group/GroupCommand.java:25]: I'll change those as well

**Pull request merged**